<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CannyEdge</class>
 <widget class="QMainWindow" name="CannyEdge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>414</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>996</width>
    <height>414</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>996</width>
    <height>414</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Canny Edge Detection</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="inputVid">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>230</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputVid">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>280</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>241</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Convert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(138, 226, 52)
}</string>
    </property>
    <property name="text">
     <string>Start Conversion</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="ThreshLow">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>48</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>31</height>
     </size>
    </property>
   </widget>
   <widget class="QSpinBox" name="ThreshHigh">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Save As</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Open">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>Threshold :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>211</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>Select Input Video :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>Output Destination :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Abort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(136, 138, 133)
}</string>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="previewOriginal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="previewEdited">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>611</width>
      <height>341</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>611</width>
      <height>341</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>611</width>
      <height>341</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>67</width>
      <height>17</height>
     </size>
    </property>
    <property name="text">
     <string>Preview:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Preview">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="About">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>7</number>
    </property>
    <property name="midLineWidth">
     <number>8</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#2e3436;&quot;&gt;Edge Detector&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#555753;&quot;&gt;v. 2.3&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#555753;&quot;&gt;https://github.com/Takundei/Canny-Edge-Detection&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; color:#2e3436;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#2e3436;&quot;&gt;- Displays edges in a video using Canny's edge detection algorithm .&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#2e3436;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#2e3436;&quot;&gt;- Please note: detecting the edges might&lt;/span&gt;&lt;span style=&quot; color:#2e3436; vertical-align:super;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; color:#2e3436;&quot;&gt; take a lot of time,however,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#2e3436;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#2e3436;&quot;&gt;- Clicking &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#555753;&quot;&gt;Abort&lt;/span&gt;&lt;span style=&quot; color:#2e3436;&quot;&gt; will save the current progress in converting the video.&lt;/span&gt;&lt;/p&gt;
&lt;p dir='rtl' style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#2e3436;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#c4a000;&quot;&gt;Tip: Try Theshold values in ratios 1:2 or 1:3&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#888a85; vertical-align:super;&quot;&gt;*&lt;/span&gt;&lt;span style=&quot; font-size:9pt; color:#888a85;&quot;&gt;will&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
